public/language/en-US/categorys.json

```diff
@@ -0,0 +1,22 @@
+{
+    "announcements": {
+        "name": "Announcements",
+        "description": "Announcements regarding our community",
+        "descriptionParsed": "Announcements regarding our community"
+    },
+    "general_discussion": {
+        "name": "General Discussion",
+        "description": "A place to talk about whatever you want",
+        "descriptionParsed": "A place to talk about whatever you want"
+    },
+    "blogs": {
+        "name": "Blogs",
+        "description": "Blog posts from individual members",
+        "descriptionParsed": "Blog posts from individual members"
+    },
+    "comments_feedback": {
+        "name": "Comments & Feedback",
+        "description": "Got a question? Ask away!",
+        "descriptionParsed": "Got a question? Ask away!"
+    }
+}
```

Comment:
Unless I'm mistaken, nested JSON strings won't work for Transifex. They work for NodeBB, but Transifex is just a single flat key/value store, unless things have changed on their end.

So you'll want to use the format like this:

```
{
    "announcements.name": "Announcements",
    ...
}
```